library/src/main/java/com/novoda/downloadmanager/lib/DefaultNotificationInitialiser.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.downloadmanager.lib;
+
+import android.app.NotificationManager;
+import android.content.Context;
+import android.support.v4.app.NotificationCompat;
+
+import com.novoda.downloadmanager.notifications.NotificationInitialiser;
+
+class DefaultNotificationInitialiser implements NotificationInitialiser {
```

Comment:
In the next PR, when we reintroduce Android O, there will be an implementation of this interface that modifies something on the `NotificationManager`, so it will do more than just create a `Builder` object. Also having both `Builder` and `Creator` in the name looks confusing 😜